x/emissions/keeper/keeper.go

```diff
@@ -1419,23 +1419,22 @@ func (k *Keeper) DripTopicFeeRevenue(ctx context.Context, topicId TopicId, block
 	if err != nil {
 		return err
 	}
-	newTopicFeeRevenue := types.TopicFeeRevenue{
-		Epoch:   block,
-		Revenue: cosmosMath.ZeroInt(),
-	}
+
 	moduleParams, err := k.GetParams(ctx)
 	if err != nil {
 		return err
 	}
 	epsilon := moduleParams.Epsilon
 	topicFeeRevenueDecayRate := moduleParams.TopicFeeRevenueDecayRate
-	topicFeeRevenueDec, err := alloraMath.NewDecFromSdkInt(topicFeeRevenue.Revenue)
+
+	topicFeeRevenueDec, err := alloraMath.NewDecFromSdkInt(topicFeeRevenue)
```

Comment:
By the way, I think I this "pattern" could be the source of other `nil dereference` errors: the `err` is not checked to be nil (!). 

So in case of error, the returned object may be nil, hence triggering a nil dereference in our face. 
Can you please check this in other parts of the changes of the new synth especially?